api/apps/api/src/modules/access-control/scenarios-acl/scenario-acl.service.ts

```diff
@@ -104,4 +108,54 @@ export class ScenarioAclService implements ScenarioAccessControl {
     });
     return otherOwnersInScenario >= 1;
   }
+
+  async updateUserInScenario(
+    scenarioId: string,
+    userAndRoleToChange: UserRoleInScenarioDto,
+    loggedUserId: string,
+  ): Promise<Either<Permit, void>> {
```

Comment:
left has Permit type, but you only use it with false, so maybe some stricter typing may be used